tests/base.py

```diff
@@ -148,6 +150,43 @@ def clear_tables(self, table_names=ALL_TABLE_NAMES_TO_CLEAR):
     def random_string_generator(size=6, chars=string.ascii_uppercase + string.digits):
         return ''.join(random.choice(chars) for x in range(size))
 
+    #@staticmethod # TODO is this needed?
+    def random_decimal_generator(self):
+        # Refactor per PR review
+        # List boundry cases (20)
+        # max exponent with max precision and max digit magnitute 0.999...e126 (pos and neg)
+        # min exponent with max precision and max digit magnitute 0.999...e-128 (pos and neg)
+        # TODO card out test overflow of above two cases?
```

Comment:
Could you link the card for this  get linked here so future testers can search for the ticket id to reproduce this issue

Reply:
Let's discuss.  Are you referring to the decimal.Rounded card or a card specific to the precision 38 with e-128?

Reply:
the ticket is linked now resolving